tests/cisco_ios/show_cdp_neighbors/cisco_ios_show_cdp_neighbors_4.yml

```diff
@@ -0,0 +1,7 @@
+---
+parsed_sample:
+  - neighbor: "acc-sw101"
+    local_interface: ""
```

Comment:
Shouldn't you be capturing "Ten 3/7" here?

Reply:
how did the test pass?